2019Labs/RHELSecurityLab/documentation/lab1_OpenSCAP.adoc

```diff
@@ -142,71 +171,90 @@ image:images/lab1.1-labuserlogin.png[300,300]
-Putting the machine into compliance (for example by changing its configuration) is called *remediation* in the SCAP terminology. Remediation changes the configuration of the machine and it is possible that you will lock yourself out or disable important workloads! As a result, it is best practice to test the remediation changes before deploying.
+Putting the machine into compliance (for example by changing its configuration) is called *remediation* in the SCAP terminology.
+As remediation changes configuration of the machine to restrict it's capabilities, it is possible that you will lock yourself out or disable workloads important to you.
+As a result, it is best practice to test the remediation and it's effects before deploying.
 
 . If not already there, log into to the workstation bastion host as *lab-user* from your desktop system *replacing GUID with your lab's GUID*. Use the password *r3dh4t1!*
 +
-[source]
 ----
 [localhost ~]$ ssh lab-user@workstation-GUID.rhpds.opentlc.com
 ----
 . Log into the *openscap.example.com* host as *root*.
 +
-[source]
 ----
 [lab-user@workstation-GUID ~]$ ssh root@openscap.example.com
 ----
 
-. All remediations will be executed on the *openscap.example.com* host. You will not make modifications to any other hosts, including the *workstation.example.com* host.
+. All remediations will be executed on the *openscap.example.com* host.
+You will not make modifications to any other hosts, including the *workstation.example.com* host.
 
-. Let's generate an Ansible playbook that will put the *openscap.example.com* machine into compliance. Let's go ahead and generate a playbook from the results:
-* use --fix-type ansible to request an ansible playbook with the fixes
+. Let's generate an Ansible playbook that will put the *openscap.example.com* machine into compliance.
+Let's go ahead and generate a playbook from the results:
 +
+Use the `--fix-type ansible` option to request an ansible playbook with the fixes:
++
+----
  [root@openscap]# oscap xccdf generate fix --fix-type ansible --result-id "" /tmp/arf.xml > playbook.yml
+----
 
-. (Optional) Generate bash remediation script and run it on target machine(s). This can be accomplished by running:
-* use --fix-type bash to request a bash script with the fixes
-* after the script is generated change its permissions so that we can run it
+. (Optional) Generate bash remediation script.
+This can be accomplished by running:
+* use `--fix-type bash` to request a bash script with the fixes
 +
+----
  [root@openscap]# oscap xccdf generate fix --fix-type bash --result-id "" /tmp/arf.xml > bash-fix.sh
- [root@openscap]# chmod +x bash-fix.sh
- [root@openscap]# ./bash-fix.sh
+----
+. By running this generated script, machine would be also put close to the compliance.
 
-. Notice that in both cases we are using empty result-id. This is a trick to avoid specifying the full result ID.
+. Notice that in both cases we are using empty `--result-id`.
```

Comment:
I would go into a somehow greater detail - i.e. oscap supports generation of fixes from a result file that has results from multiple benchmarks (correct me if I am wrong). However, as there is only one result, we specify the empty result ID, and that's good enough.

Reply:
When YOU are not sure about when multiple results in the result file happen, do you think it's necessary to explain the point during showcasing of capabilities? :)